src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+</Callout>
+
+Setting a proxy URL for the browser agent can be valuable because it can help you to get around ad blockers and security VPNs that block client requests to third-party domains.
+
+Ad blockers and security VPNs often block third-party domains because they can be used to track users across the web. By proxying your requests through a first-party subdomain, you can bypass these blockers and ensure that your observability data is still being collected.
+
+## How proxies work [#how-proxies-work]
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, you can set up a proxy by configuring the following browser agent properties:
+* `init.proxy.assets` determines where to fetch the browser agent.
+  * If you’re using the copy/paste installation method, you’ll need to manually edit your JavaScript snippet and set `assets` to your domain URL. For example, if you’re using browser agent v1.240.0 Pro+SPA, and you set `init.proxy.assets` to `assets.mydomain.com`, the browser agent will be loaded from `https://assets.mydomain.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.
+  * If you’re using the NPM installation method, this configuration is not relevant. NPM-installed browser agents undergo the same bundling process as the host application and should result in agent assets in the `output` directory alongside the host application. When the host application is deployed, those agent assets should be deployed as well resulting in agent code already loading from the same domain as the site.
+* `init.proxy.beacon` determines where analytics data gets sent.
+  * For both installation methods, you’ll need to set `beacon` to your domain URL.
+  * For example, if you set `beacon` to `beacon.mydomain.com`, all observability data will be sent to `https://beacon.mydomain.com/YOUR_NR_ENDPOINT`.
+
+If you’re concerned about user privacy, you can exclude the `X-Forwarded-For HTTP` header, which is used to geolocate clients. If you exclude it, New Relic will geolocate your proxy as the client instead.
+
+### Recommendations for proxy URLs
+
+When configuring these properties, we recommend the following:
+
+* Use a first-party subdomain. Even though you can technically use a third-party subdomain, it’s likely to get blocked by ad blockers and security VPNs.
+* The subdomain URL pattern is `<domain>[:<port>][/<path>]`. Make sure to omit the protocol `https://` and ending URL slash. The `port` and `path` are optional and will be respected if added.
+* If you’re configuring both `assets` and  `beacon`, set different values for each proxy URL, such as using a different subdomain for each URL or one that contains a different path. This will make configuring the proxy server a lot easier.
+
+## Set up your proxy [#set-up]
+
+To set up your proxy, see the instructions below based on which deployment method you used to install the browser agent.
+
```

Comment:
```suggestion
## Set up your proxy [#set-up]

To set up your proxy, see the instructions below based on which deployment method you used to install the browser agent.

### Recommendations

When configuring these properties, we recommend the following:

* Use a first-party subdomain. Even though you can technically use a third-party subdomain, it’s likely to get blocked by ad blockers and security VPNs.
* The subdomain URL pattern is `<domain>[:<port>][/<path>]`. Make sure to omit the protocol `https://` and ending URL slash. The `port` and `path` are optional and will be respected if added.
* If you’re configuring both `assets` and  `beacon`, set different values for each proxy URL, such as using a different subdomain for each URL or one that contains a different path. This will make configuring the proxy server a lot easier.

```